sycl/include/CL/sycl/device.hpp

```diff
@@ -161,17 +161,18 @@ class __SYCL_EXPORT device {
   ///
   /// \param extension_name is a name of queried extension.
   /// \return true if SYCL device supports the extension.
-  bool has_extension(const string_class &extension_name) const;
+  __SYCL2020_DEPRECATED("use device aspects APIs instead")
```

Comment:
better to give the right name in the message, is it "has_aspect"?

Reply:
It is just `has`.